### ISSUE TITLE
HTML class - not producing the expected results, on Windows 7

### ISSUE THREAD

**Issue by ReddyKilowatt** [prompt-toolkit/python-prompt-toolkit#775]
Using prompt_toolkit 2.07, with Python 3.6.6 (32-Bit) on Windows 7 Pro, 64-Bit 

[https://python-prompt-toolkit.readthedocs.io/en/stable/pages/printing_text.html#html](url)

refers to this code

`from __future__ import unicode_literals, print_function
from prompt_toolkit import print_formatted_text, HTML

print_formatted_text(HTML('<b>This is bold</b>'))
print_formatted_text(HTML('<i>This is italic</i>'))
print_formatted_text(HTML('<u>This is underlined</u>'))`

However, only plain text is displayed. Bold, italics, and underlined are not seen

In other words, the code aboce produces the same result as the code below.

`print_formatted_text('This is bold')
print_formatted_text('This is italic')
print_formatted_text('This is underlined')
`

**Comment by jonathanslenders** [comment]
@ReddyKilowatt: So, for Windows 7, I think both bold and italic are not supported by the operating system.

Underline is not supported, but could work in theory using the ``COMMON_LVB_UNDERSCORE `` console attribute. However, that only works for the DBCS code page, and I don't know whether we'll support this.

See:
https://stackoverflow.com/questions/29386552/how-to-underline-text-using-printf-in-c
https://docs.microsoft.com/en-us/windows/console/char-info-str
https://docs.microsoft.com/en-us/windows/desktop/Intl/double-byte-character-sets

**Comment by ReddyKilowatt** [comment]
ok, I understand some features will not worked depending on the OS, it would be helpful to add this to the documentation, or remove all of the features that are not supported, for each platform.

Thanks

**Comment by jvickroy** [comment]
I am encountering the exact same issues on Windows 10 with Python 3.7.  As previously stated, it would be preferable for the documentation to state these deficiencies.  Otherwise, I do wish to thank the authors for this package.

**Comment by Samuel38** [comment]
I have a similar issue on Windows 10 Family 20H2 and Python 3.9.5. 
 - Italic doesn't show at all
 - Underline only shows once the program is closed. While it's running, it shows as normal text
 - Bold and colors work fine.
